ansible/roles/controller/tasks/deploy.yml

```diff
@@ -125,6 +125,11 @@
       "CONFIG_whisk_activation_payload_max": "{{ limit_activation_payload | default() }}"
 
       "RUNTIMES_MANIFEST": "{{ runtimesManifest | to_json }}"
+      "CONFIG_whisk_runtimes_defaultImagePrefix": "{{ runtimes_default_image_prefix | default() }}"
+      "CONFIG_whisk_runtimes_defaultImageTag": "{{ runtimes_default_image_tag | default() }}"
+      "CONFIG_whisk_runtimes_bypassPullForLocalImages": "{{ runtimes_bypass_pull_for_local_images | default() }}"
```

Comment:
fwiw we don't actually need this setting in the controller; ok to keep it though.

Reply:
yeah, wasn't sure what the best policy was here.  Seems like a latent bug to not wire up the overrides (just in case some later change needs them), but if we don't consistently do that for other flags then I could remove these.